src/signals/IncidentMap/components/AddressLocation/AdressLocation.test.tsx

```diff
@@ -1,61 +1,117 @@
 
 import type { Props } from './AddressLocation'
 import { AddressLocation } from './AddressLocation'
 
-const coords = {
-  lat: 52.3731081,
-  lng: 4.8932945,
-}
-
-const mockPdokAddress = {
-  huisnummer: '178',
-  openbare_ruimte: 'Warmoesstraat',
-  postcode: '1012JK',
+const mockAddress = {
+  postcode: '1000 AA',
+  huisnummer: '100',
   woonplaats: 'Amsterdam',
+  openbare_ruimte: 'West',
 }
 
-const mockPdokResponse = {
-  id: 1,
-  value: 'mockValue',
+const mockList = (props: ReactPropTypes) => (
+  <ul className="suggestList" {...props}>
+    <li>Suggestion #1</li>
+    <li>Suggestion #2</li>
+  </ul>
+)
+
+const mockPDOKResponse: PdokResponse = {
+  id: 'foo',
+  value: 'Zork',
   data: {
-    location: coords,
-    address: mockPdokAddress,
+    location: {
+      lat: 12.282,
+      lng: 3.141,
+    },
+    address: mockAddress,
   },
 }
 
-jest.mock('./styled', () => ({
-  ...jest.requireActual('./styled'),
-  StyledPDOKAutoSuggest: ({ onSelect, value, ...rest }: any) => {
-    onSelect(mockPdokResponse)
-    return <div {...rest}>{value}</div>
-  },
-}))
+jest.mock(
+  'components/PDOKAutoSuggest/PDOKAutoSuggest',
+  () =>
+    ({
+      className,
+      onSelect,
+      value,
+      onClear,
+      onFocus,
+      onData,
+    }: PDOKAutoSuggestProps) =>
+      (
+        <span data-testid="pdokAutoSuggest" className={className}>
+          <button data-testid="autoSuggestClear" onClick={onClear}>
+            Clear input
+          </button>
+          <button onClick={() => onSelect(mockPDOKResponse)}>selectItem</button>
+          <button
+            data-testid="getDataMockButton"
+            type="button"
+            onClick={() => {
+              onData && onData(mockList)
+            }}
+          />
+          <input data-testid="autoSuggestInput" type="text" onFocus={onFocus} />
+          <span>{value}</span>
+        </span>
+      )
+)
 
 const defaultProps: Props = {
   setCoordinates: jest.fn(),
-  address: mockPdokAddress,
-  setAddress: jest.fn(),
 }
 
 describe('AddresLocation', () => {
   it('tests whether the onAddressSelect actually sets the coordinates', async () => {
     render(<AddressLocation {...defaultProps} />)
 
+    userEvent.click(screen.getByText('selectItem'))
+
     //the mocked PDOKAutoSuggest triggers automatically onSelect/onAddressSelect, no userEvents are necessary
-    expect(defaultProps.setCoordinates).toHaveBeenCalledWith(coords)
+    expect(defaultProps.setCoordinates).toHaveBeenCalledWith(
+      mockPDOKResponse.data.location
+    )
   })
 
   it('shows the address in the searchbar', () => {
+    const mockPdokAddress = {
+      huisnummer: '178',
+      openbare_ruimte: 'Warmoesstraat',
+      postcode: '1012JK',
+      woonplaats: 'Amsterdam',
+    }
+
+    const props = {
+      ...defaultProps,
+      address: mockPdokAddress,
+    }
+    render(<AddressLocation {...props} />)
+
+    const input = screen.getByText('Warmoesstraat 178, 1012JK Amsterdam')
```

Comment:
unnecessary declaration, can put it in the expect

Reply:
I like it for readability...